src/functTest/groovy/com/bmuschko/gradle/docker/DockerWorkflowFunctionalTest.groovy

```diff
@@ -429,6 +429,61 @@ class DockerWorkflowFunctionalTest extends AbstractGroovyDslFunctionalTest {
         build('workflow')
     }
 
+    def "Can build an image, create a container and add/drop capabilities"() {
```

Comment:
This test should go into `DockerCreateContainerFunctionalTest`. I want to get rid of `DockerWorkflowFunctionalTest.groovy` long-term.

Reply:
Another tests seems to be failing with your change on CI. Could you have a look? I am not 100% sure if they are related.

Reply:
Moved the test, and then that other one failing is just a change in output, I'll do a quick separate PR for that just cause